protocol/communities/community.go

```diff
@@ -1605,6 +1606,11 @@ func (o *Community) CanPost(pk *ecdsa.PublicKey, chatID string, grantBytes []byt
 		return false, nil
 	}
 
+	// this resolves the issue, however, I'm not sure how it should play with grants and tokens below
+	if !o.isMember() {
```

Comment:
That looks right. We don't need to check the tokens or whatever if this condition doesn't pass. If someone is not a member, they cannot post.